core/new-gui/e2e/backend-config.yml

```diff
@@ -0,0 +1,10 @@
+server:
```

Comment:
Is there any specific benefit to use a new backend config file rather than the original config? My concern is that in future the backend might need more config file to run,  such as a config file for sql username and password.